### DISCUSSION TITLE
Scoping renderhooks to auth pages

### DISCUSSION THREAD

**Discussion by underdpt** [filamentphp/filament#8212] [Help]
Hello,

I'm trying to show the plugin [Language Switch](https://filamentphp.com/plugins/bezhansalleh-language-switch) on the auth pages (login, password reset, etc.).

This is how I'm doing it with a renderhook attached to the panel:

```php
->renderHook(
    name: 'panels::body.start',
    hook: function (): string {
        return Blade::render(
            <<<'EOD'
            <div class="absolute top-2 right-2">
                @livewire('switch-filament-language')
            </div>
            EOD
        );
    },
    scopes: [
        Pages\Auth\Login::class,
        Pages\Auth\Register::class,
        Pages\Auth\EmailVerification\EmailVerificationPrompt::class,
        Pages\Auth\PasswordReset\RequestPasswordReset::class,
        Pages\Auth\PasswordReset\ResetPassword::class,
    ]
);
```
The component doesn't show on the auth pages until I fully remove the `scopes` parameter. It seems that scoping doesn't work for the auth pages, could that be added?

**Comment by danharrin** [comment]
It would require a new feature to be added to the core, I will convert this into an issue so it is added to our roadmap

**Comment by danharrin** [comment]
#11119
